ui/src/View.elm

```diff
@@ -451,6 +427,42 @@ spinner =
     div [ class "loader" ] []
 
 
+facetSelector : String -> Int -> List String -> (String -> NewFilter) -> List Facet -> Html Msg
+facetSelector title total selectedValues filterMsg facets =
+    let
+        choice entry =
+            let
+                active =
+                    List.member entry.value selectedValues
+
+                countInfo =
+                    " (" ++ (toString entry.count) ++ ")"
+
+                msg =
+                    UpdateFilter (filterMsg entry.value)
+            in
+                div [ class "checkbox" ]
+                    [ label []
+                        [ input
+                            [ type_ "checkbox"
+                            , value entry.value
+                            , checked active
+                            , onWithOptions
+                                "click"
+                                { preventDefault = True, stopPropagation = False }
```

Comment:
is this needed? I don't remember having to deal with that in the past, I simply used a "onClick Msg"